vault/resource_saml_auth_backend.go

```diff
@@ -221,6 +235,24 @@ func samlAuthBackendRead(ctx context.Context, d *schema.ResourceData, meta inter
 		}
 	}
 
+	log.Printf("[DEBUG] Reading saml auth tune from %q", id+"/tune")
+	rawTune, err := authMountTuneGet(ctx, client, "auth/"+id)
```

Comment:
Does this require a Vault policy change? If yes, we should make sure to update the changelog accordingly and specifically call out a behavior change. See https://github.com/hashicorp/terraform-provider-vault/blob/main/CHANGELOG.md#500-may-21-2025 as an example.